controllers/controllers/resource/template.go

```diff
@@ -38,6 +39,12 @@ type VsphereTemplate struct {
 	now anywhereTypes.NowFunc
 }
 
+type CloudStackTemplate struct {
+	ResourceFetcher
```

Comment:
Why are these embedded? I'm not sure why a `CloudStackTemplate` would provide fetching behavior?

Jumping the gun: if its because the other constructs do this, ignore the other constructs.

Reply:
Created https://github.com/aws/eks-anywhere/issues/1645